packages/react-native-session-replay/ios/Sources/DdSessionReplayImplementation.swift

```diff
@@ -10,6 +10,11 @@ import DatadogInternal
 import DatadogSDKReactNative
 import React
 
+struct SVGData: Codable {
```

Comment:
nit: if only used internally + you could also add some doc 
```suggestion
internal struct SVGData: Codable {
```